test/acceptance/features/steps/steps.py

```diff
@@ -278,6 +278,43 @@ def then_envFrom_contains_intermediate_secret_name(context, intermediate_secret_
         intermediate_secret_name, wait=True, timeout=120) is not None, f"There is no deployment with intermediate secret {intermediate_secret_name}"
 
 
+# STEP
+@given(u'OLM Operator Backend is running')
+def given_olm_operator_backend(context):
+    openshift = Openshift()
+    _ = openshift.oc_apply_yaml_file(os.path.join(os.getcwd(), "test/acceptance/resources/backend.operator.manifest.yaml"))
```

Comment:
You can use regex to as we used in other steps to verify if oc apply is successful. [Similar regex you can use.](https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/steps/steps.py#L292)

Reply:
@pratikjagrut that regex will work only for a yaml file having a particular kind. I have 2 yamls in one file

Reply:
@pratikjagrut @pmacik perhaps `oc wait` could help us here?

Reply:
I have added an exit code var in the oc_apply_yaml_file function. PTAL

Reply:
It looks good to me.